packages/block-library/src/media-text/editor.scss

```diff
@@ -59,18 +59,6 @@
 	max-width: unset;
 }
 
-figure.block-library-media-text__media-container {
-	margin: 0;
-	height: 100%;
-	width: 100%;
-}
-
-.wp-block-media-text .block-library-media-text__media-container img,
-.wp-block-media-text .block-library-media-text__media-container video {
-	vertical-align: middle;
-	width: 100%;
-}
-
```

Comment:
Were these expected to be removed? Looking at this, I would expect it should just be updated to use `wp-block-` instead of `block-library-`:

https://github.com/WordPress/gutenberg/pull/20439/files#diff-b3343813f22b1e93da7b15d53d8fd50b

Reply:
Hi @aduth they were removed as part of https://github.com/WordPress/gutenberg/pull/20539.